crime-applications-adaptor/application/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/ApplicantMapper.java

```diff
@@ -80,12 +86,33 @@ private EmploymentStatus mapEmploymentStatus(MaatApplicationExternal crimeApplyR
     return employmentStatus;
   }
 
-  private PartnerContraryInterest mapPartnerContraryInterest() {
+  private PartnerContraryInterest mapPartnerContraryInterest(
+      MaatApplicationExternal crimeApplyResponse) {
     PartnerContraryInterest partnerContraryInterest = new PartnerContraryInterest();
-    partnerContraryInterest.setCode(PARTNER_CONTRARY_INTEREST_CODE_DEFAULT_NOCON);
+    Partner partner = crimeApplyResponse.getClientDetails().getPartner();
+    if (Objects.nonNull(partner)) {
+      mapContraryInterest(partner, partnerContraryInterest);
+    } else {
+      partnerContraryInterest.setCode(PARTNER_CONTRARY_INTEREST_CODE_DEFAULT_NOCON);
+    }
     return partnerContraryInterest;
   }
 
+  private void mapContraryInterest(
```

Comment:
Would consider renaming this to `mapContraryInterestCode` as it only sets the code.

Reply:
done